Events/src/main/java/com/github/swcraftlyon/meetup/infra/ApplicationConfig.java

```diff
@@ -0,0 +1,32 @@
+        System.out.println("M::::::M M::::M M::::M M::::::Me::::::e     e:::::ee::::::e     e:::::etttttt:::::::tttttt    u::::u    u::::u pp::::::ppppp::::::p     !:::!");
+        System.out.println("M::::::M  M::::M::::M  M::::::Me:::::::eeeee::::::ee:::::::eeeee::::::e      t:::::t          u::::u    u::::u  p:::::p     p:::::p     !:::!");
+        System.out.println("M::::::M   M:::::::M   M::::::Me:::::::::::::::::e e:::::::::::::::::e       t:::::t          u::::u    u::::u  p:::::p     p:::::p     !:::!");
+        System.out.println("M::::::M    M:::::M    M::::::Me::::::eeeeeeeeeee  e::::::eeeeeeeeeee        t:::::t          u::::u    u::::u  p:::::p     p:::::p     !!:!!");
+        System.out.println("M::::::M     MMMMM     M::::::Me:::::::e           e:::::::e                 t:::::t    ttttttu:::::uuuu:::::u  p:::::p    p::::::p      !!! ");
+        System.out.println("M::::::M               M::::::Me::::::::e          e::::::::e                t::::::tttt:::::tu:::::::::::::::uup:::::ppppp:::::::p          ");
+        System.out.println("M::::::M               M::::::M e::::::::eeeeeeee   e::::::::eeeeeeee        tt::::::::::::::t u:::::::::::::::up::::::::::::::::p       !!! ");
+        System.out.println("M::::::M               M::::::M  ee:::::::::::::e    ee:::::::::::::e          tt:::::::::::tt  uu::::::::uu:::up::::::::::::::pp       !!:!!");
+        System.out.println("MMMMMMMM               MMMMMMMM    eeeeeeeeeeeeee      eeeeeeeeeeeeee            ttttttttttt      uuuuuuuu  uuuup::::::pppppppp          !!! ");
+        System.out.println("                                                                                                                p:::::p                      ");
+        System.out.println("                                                                                                                p:::::p                      ");
+        System.out.println("                                                                                                               p:::::::p                     ");
+        System.out.println("                                                                                                               p:::::::p                     ");
+        System.out.println("                                                                                                               p:::::::p                     ");
+        System.out.println("                                                                                                               ppppppppp                     ");
+        System.out.println("                                                                                                                                             ");
+        System.out.println("Welecom to your new meetup application ! Lets meetup ...");
```

Comment:
Que pensez vous de ne pas mettre de banner ou juste une ligne et/ou de
l'afficher par un logger configurable histoire de ne pas polluer la sortie
standard ?

Le ven. 15 avr. 2016 12:07, Jean Detoeuf <notifications@github.com> a
écrit :

> In
> Events/src/main/java/com/github/swcraftlyon/meetup/infra/ApplicationConfig.java
> <https://github.com/swcraftlyon/meetup/pull/29#discussion_r59853482>:
>
> > +        System.out.println("M::::::M  M::::M::::M  M::::::Me:::::::eeeee::::::ee:::::::eeeee::::::e      t:::::t          u::::u    u::::u  p:::::p     p:::::p     !:::!");
> > +        System.out.println("M::::::M   M:::::::M   M::::::Me:::::::::::::::::e e:::::::::::::::::e       t:::::t          u::::u    u::::u  p:::::p     p:::::p     !:::!");
> > +        System.out.println("M::::::M    M:::::M    M::::::Me::::::eeeeeeeeeee  e::::::eeeeeeeeeee        t:::::t          u::::u    u::::u  p:::::p     p:::::p     !!:!!");
> > +        System.out.println("M::::::M     MMMMM     M::::::Me:::::::e           e:::::::e                 t:::::t    ttttttu:::::uuuu:::::u  p:::::p    p::::::p      !!! ");
> > +        System.out.println("M::::::M               M::::::Me::::::::e          e::::::::e                t::::::tttt:::::tu:::::::::::::::uup:::::ppppp:::::::p          ");
> > +        System.out.println("M::::::M               M::::::M e::::::::eeeeeeee   e::::::::eeeeeeee        tt::::::::::::::t u:::::::::::::::up::::::::::::::::p       !!! ");
> > +        System.out.println("M::::::M               M::::::M  ee:::::::::::::e    ee:::::::::::::e          tt:::::::::::tt  uu::::::::uu:::up::::::::::::::pp       !!:!!");
> > +        System.out.println("MMMMMMMM               MMMMMMMM    eeeeeeeeeeeeee      eeeeeeeeeeeeee            ttttttttttt      uuuuuuuu  uuuup::::::pppppppp          !!! ");
> > +        System.out.println("                                                                                                                p:::::p                      ");
> > +        System.out.println("                                                                                                                p:::::p                      ");
> > +        System.out.println("                                                                                                               p:::::::p                     ");
> > +        System.out.println("                                                                                                               p:::::::p                     ");
> > +        System.out.println("                                                                                                               p:::::::p                     ");
> > +        System.out.println("                                                                                                               ppppppppp                     ");
> > +        System.out.println("                                                                                                                                             ");
> > +        System.out.println("Welecom to your new meetup application ! Lets meetup ...");
>
> Qu'est-ce que vous pensez de mettre le banner dans un fichier (comme sur
> Spring Boot). Ca serait aussi plus lisible.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/swcraftlyon/meetup/pull/29/files/3fee659009e78d3482a9e4186e82710dd9f2ea15#r59853482>
>